### PR TITLE
boop: test mostly runs, yay

### PR DESCRIPTION
**Pull request by antonbobkov**

### CODE REVIEW

File: erlang/blocktree/blocktree.erl [hunk 1]
```diff
@@ -249,7 +249,7 @@ generate_new_block(PreviousBlockId, TreeData)
      }.
     
 
-get_block_by_id(TreeData, Id)    
+get_block_by_id(Id, TreeData)
```
Comment (pdlla) [inline erlang/blocktree/blocktree.erl]: I liked the other way better (ಠ益ಠ)

Comment (pdlla) [reply erlang/blocktree/blocktree.erl]: better for function currying in haskell (╯°□°）╯︵ ┻━┻

File: erlang/pop_protocol/pop_protocol_tests.erl [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+-import(pop_protocol, [add_one_block/3, get_genesis_tree_data/1]).
+-import(blocktree, [add_new_block/2, get_block_by_id/2]).
+-import(my_crypto, [hash/1, sign/2, verify/3, read_file_key/2]).
+-import(my_serializer, [serialize_object/1]).
+
+-include_lib("stdlib/include/assert.hrl").
+
+-include("../potato_records.hrl").
+
+make_block(PrevId, PrivateKey, PublicKey, Index, Time) ->
```
Comment (pdlla) [inline erlang/pop_protocol/pop_protocol_tests.erl]: shouldn't this have hash of previous block not just id?

Comment (pdlla) [reply erlang/pop_protocol/pop_protocol_tests.erl]: or is id the hash ( ﾟдﾟ)

File: erlang/pop_protocol/pop_protocol_tests.erl [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		    },
+    Block1.
+    
+
+
+cmd() ->
+    %% make verifier array, they share keys
+    PrivateKey = my_crypto:read_file_key(private, "key1.prv"),
+    PublicKey = my_crypto:read_file_key(public, "key1.pub"),
+    VerFunc = fun(Index) -> #verifier_public_info{index = Index, public_key = PublicKey} end,
```
Comment (pdlla) [inline erlang/pop_protocol/pop_protocol_tests.erl]: lambda ( ﾟдﾟ)

File: erlang/blocktree/blocktree.erl (hunk 1 above)
Comment (antonbobkov) [reply erlang/blocktree/blocktree.erl]: ┬──┬◡ﾉ(° -°ﾉ)

File: erlang/pop_protocol/pop_protocol_tests.erl (hunk 2 above)
Comment (antonbobkov) [reply erlang/pop_protocol/pop_protocol_tests.erl]: id is the hash O:

File: erlang/blocktree/blocktree.erl (hunk 1 above)
Comment (pdlla) [reply erlang/blocktree/blocktree.erl]: ┻━┻ ︵ヽ(`Д´)ﾉ︵﻿ ┻━┻

File: erlang/pop_protocol/pop_protocol_tests.erl (hunk 2 above)
Comment (pdlla) [reply erlang/pop_protocol/pop_protocol_tests.erl]: /ᐠ｡ꞈ｡ᐟ\

Comment (antonbobkov) [reply erlang/pop_protocol/pop_protocol_tests.erl]: ✧ʕ̢̣̣̣̣̩̩̩̩·͡˔·ོɁ̡̣̣̣̣̩̩̩̩✧

File: erlang/blocktree/blocktree.erl (hunk 1 above)
Comment (antonbobkov) [reply erlang/blocktree/blocktree.erl]: ノ┬─┬ノ ︵ ( \o°o)\

Comment (pdlla) [reply erlang/blocktree/blocktree.erl]: ﾉ(° -°ﾉ)◡ﾉ┬──┬ﾉ

**Pull request merged**